internal/Invoke-SmoCheck.ps1

```diff
@@ -8,19 +8,19 @@ Checks for PowerShell SMO version vs SQL Server's SMO version.
 	[CmdletBinding()]
 	param (
 		[Parameter(Mandatory = $true)]
-		[object]$SqlServer
+		[object]$SqlInstance
 	)
 	
 	if ($script:smocheck -ne $true)
 	{
 		$script:smocheck = $true
-		$smo = (([AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.Fullname -like "Microsoft.SqlServer.SMO,*" }).FullName -Split ", ")[1]
+		$smo = (([AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.Fullname -like "Microsoft.SqlInstance.SMO,*" }).FullName -Split ", ")[1]
```

Comment:
I dont think msft changed the types around, but I might be wrong, this might need a reversion